closed/custom/modules/java.base/Copy.gmk

```diff
@@ -30,6 +30,28 @@ $(call openj9_copy_files,, \
 	$(OPENJ9_VM_BUILD_DIR)/include/jvmti.h \
 	$(INCLUDE_TARGET_DIR)/jvmti.h)
 
+ifeq ($(OPENJDK_TARGET_OS), zos)
+$(call openj9_copy_files,, \
+$(OPENJ9_TOPDIR)/runtime/include/jni_convert.h \
+$(INCLUDE_TARGET_DIR)/jni_convert.h)
+
+# Customized z/OS 31/64-bit Interoperability JNI header files.
+# These files are generated as part of OpenJ9 build.
```

Comment:
Why is 'Interoperability' capitalized?
Please add the missing 'an' in 'part of an OpenJ9 build'.